### PR TITLE
Adding coverage report to Jenkins

### PR DESCRIPTION
**Pull request by Mcthomas777**

### CODE REVIEW

**Comment by Mcthomas777** [comment]
> our changelog is now auto generated
> 
> to make keep it beauty change commit message:
> 
> ci: added report coverage on Jenkins
> 
> git commit --amend # do change text git push --force-with-lease

Done

File: Jenkinsfile [hunk 1]
```diff
@@ -114,6 +114,15 @@ pipeline
         always
         {
             junit 'reports/*.xml'
+            // Coverage report
```
Comment (sebastianpfischer) [inline Jenkinsfile:117]: Can you please just upload it as an artifact? The eclipse foundation do not accept it

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (Pog3k) [reply Jenkinsfile:117]: zip first .
archiveArtifacts artifacts: 'CoverageReport.zip', followSymlinks: false

**Review by Pog3k — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
Due to the nature of the eclipse jenkins, we are not able to store the coverage, therefore this PR should be cancelled.

**Pull request closed without merging**